### DISCUSSION TITLE
My Account Suspencion

### DISCUSSION THREAD

**Discussion by VForMaske** [community/community#77488] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Firstly sorry for my bad English, 

I forgot my account's pass. I tried three or four passwords then my account has been suspended, i created a ticket. Then they told me about verification after that i sent a mail from my verified mail and reply the ticket. After a day they say, again send a mail for verification. Me and my project teammates are waitin' for remove suspencion. I'm waitin' for two days, why the things are looping or when my suspencion gonna be removed?

**Comment by awirnr** [comment]
Response #2
